docker-compose.yml

```diff
@@ -5,11 +5,11 @@ server:
   volumes:
    - .:/src
    - /var/run/docker.sock:/tmp/docker.sock
-   - ./config.yml:/config.yml
+   - $HOME/volumes/ci/roger:/roger/
```

Comment:
This wont work for whoever doesnt have that specific folder, I would suggest you create a directory in root called `roger` so that everyone will anyhow have it, and move the config files in there :)